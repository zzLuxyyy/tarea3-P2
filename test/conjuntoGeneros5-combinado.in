crearConjuntoGeneros 10
cardinalConjuntoGeneros
cantMaxConjuntoGeneros
insertarConjuntoGeneros 0
insertarConjuntoGeneros 1
insertarConjuntoGeneros 2
borrarDeConjuntoGeneros 1
insertarConjuntoGeneros 3
insertarConjuntoGeneros 6
insertarConjuntoGeneros 7
borrarDeConjuntoGeneros 2
insertarConjuntoGeneros 8
insertarConjuntoGeneros 9
insertarConjuntoGeneros 1
insertarConjuntoGeneros 2
cardinalConjuntoGeneros
cantMaxConjuntoGeneros
imprimirConjuntoGeneros
unionConjuntoGeneros 5 1 3 4 5 6 
interseccionConjuntoGeneros 5 1 3 4 5 6
diferenciaConjuntoGeneros 5 1 3 4 5 6
borrarDeConjuntoGeneros 1
borrarDeConjuntoGeneros 2
unionConjuntoGeneros 5 1 3 4 5 6 
interseccionConjuntoGeneros 5 1 3 4 5 6
diferenciaConjuntoGeneros 5 1 3 4 5 6
borrarDeConjuntoGeneros 8
borrarDeConjuntoGeneros 9
unionConjuntoGeneros 5 1 3 4 5 6 
interseccionConjuntoGeneros 5 1 3 4 5 6
diferenciaConjuntoGeneros 5 1 3 4 5 6
liberarConjuntoGeneros
Fin