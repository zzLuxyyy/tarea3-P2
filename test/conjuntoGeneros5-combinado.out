1>Se creó conjunto generos con cantMax 10 exitosamente.
2>El cardinal del conjunto es: 0.
3>La cantidad máxima del conjunto es: 10.
4>Se ejecutó 'insertarTConjuntoGeneros con genero de id 0 exitosamente.
5>Se ejecutó 'insertarTConjuntoGeneros con genero de id 1 exitosamente.
6>Se ejecutó 'insertarTConjuntoGeneros con genero de id 2 exitosamente.
7>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 1 exitosamente.
8>Se ejecutó 'insertarTConjuntoGeneros con genero de id 3 exitosamente.
9>Se ejecutó 'insertarTConjuntoGeneros con genero de id 6 exitosamente.
10>Se ejecutó 'insertarTConjuntoGeneros con genero de id 7 exitosamente.
11>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 2 exitosamente.
12>Se ejecutó 'insertarTConjuntoGeneros con genero de id 8 exitosamente.
13>Se ejecutó 'insertarTConjuntoGeneros con genero de id 9 exitosamente.
14>Se ejecutó 'insertarTConjuntoGeneros con genero de id 1 exitosamente.
15>Se ejecutó 'insertarTConjuntoGeneros con genero de id 2 exitosamente.
16>El cardinal del conjunto es: 8.
17>La cantidad máxima del conjunto es: 10.
18>Conjunto de generos:
0 1 2 3 6 7 8 9
19>Conjunto de generos unión:
0 1 2 3 4 5 6 7 8 9
20>Conjunto de generos intersección:
1 3 6
21>Conjunto de generos diferencia:
0 2 7 8 9
22>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 1 exitosamente.
23>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 2 exitosamente.
24>Conjunto de generos unión:
0 1 3 4 5 6 7 8 9
25>Conjunto de generos intersección:
3 6
26>Conjunto de generos diferencia:
0 7 8 9
27>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 8 exitosamente.
28>Se ejecutó 'borrarDeTConjuntoGeneros con genero de id 9 exitosamente.
29>Conjunto de generos unión:
0 1 3 4 5 6 7
30>Conjunto de generos intersección:
3 6
31>Conjunto de generos diferencia:
0 7
32>Se ejecutó 'liberarTConjuntoGeneros' exitosamente.
33>Fin.
